cherab/openadas/openadas.py

```diff
@@ -134,20 +134,20 @@ def beam_cx_pec(self, donor_ion, receiver_ion, receiver_charge, transition):
         if isinstance(donor_ion, Isotope):
             donor_ion = donor_ion.element
 
-        if isinstance(receiver_ion, Isotope):
-            receiver_ion = receiver_ion.element
+        receiver_ion_element = receiver_ion.element if isinstance(receiver_ion, Isotope) else receiver_ion
```

Comment:
Could you please add a comment that you are extracting here the element of the passed isotope of something of the sort? It would be great if the code flow was a little bit more commented, so later we have easier code reading.